composer.json

```diff
@@ -84,7 +84,8 @@
     "ext-intl": "*",
     "pear/mail_mime": "~1.10",
     "pear/db": "1.10",
-    "civicrm/composer-compile-lib": "~0.3 || ~1.0"
+    "civicrm/composer-compile-lib": "~0.3 || ~1.0",
+    "ext-json": "*"
```

Comment:
@eileenmcnaughton was this intentional?

Reply:
@seamuslee001 yep - per pr template  - it communicates to the IDE that it is present. No impact on anything else